docs/sources/dashboards/create-reports/_index.md

```diff
@@ -0,0 +1,331 @@
+
+## Requirements
+
+- SMTP must be configured for reports to be sent. Refer to [SMTP](ref:smtp) in [Configuration](ref:configuration) for more information.
+- The Image Renderer plugin must be installed or the remote rendering service must be set up. Refer to [Image rendering](ref:image-rendering) for more information.
+
+## Access control
+
+When [RBAC](ref:rbac) is enabled, you need to have the relevant [Permissions](ref:permission) to create and manage reports.
+
+## Create a report
+
+Only organization administrators can create reports by default. You can customize who can create reports with [Role-based access control](ref:role-based-access-control).
+
+The report creation process is multi-step, but you don't need to complete these steps in order and you can skip steps by clicking a step name at the top of the page:
+
+![Reporting wizard](/media/docs/grafana/dashboards/screenshot-reporting-wizard-v11.5.png)
+
+To create a report, follow these steps:
+
+1. Click **Dashboards > Reports** in the main menu.
+1. Click **+ Create a new report**.
+1. Complete the report steps, as needed; you don't need to complete these steps in order and you can skip steps by clicking a step name at the top of the page:
+   - [Select dashboard](#1-select-dashboard)
+   - [Format report](#2-format-report)
+   - [Schedule](#3-schedule)
+   - [Share](#4-share)
+   - [Confirm](#5-confirm)
+1. Click one of the following buttons in the top-right corner of the screen:
+   - **Send** or **Schedule send** - The report is sent according the schedule you've set.
+   - **Save as draft** - You can save a draft at any point during the report creation or update process, even if it's missing required fields. The report won't be sent according to its schedule while it's a draft.
+   - **Discard** - Delete the report draft. This action can't be reversed.
+
+### 1. Select dashboard
+
+At this step, select the dashboard or dashboards on which the report is based, as well as the variables and time range for those dashboards. The options are:
+
+<!-- prettier-ignore-start -->
+
+| Option | Description |
+| ------ | ----------- |
+| Source dashboard | Select the dashboard from which you want to generate the report. |
+| [Template variables](#template-variables) | Select the variable **Datasource** and **Instance** for the selected dashboard. This option is only displayed if the dashboard has variables. |
+| [Time range (optional)](#time-range) | By default, reports use the saved time range of the dashboard. Optionally, you can change the time range of the report. |
+| Add another dashboard | Add more dashboards to the report. Additional dashboards will be rendered as new pages in the same PDF file, or additional images if you chose to embed images in your report email. You can't add the same dashboard to a report multiple times. |
```

Comment:
> You can’t add the same dashboard to a report multiple times

This is not true, a Report can contain the same dashboard more than once. One possible use case could be with different time ranges for example.